src/lib/intersection.js

```diff
@@ -0,0 +1,16 @@
+const intersection = (array1,array2) => {
```

Comment:
this method still needs a test before this branch is merged. also, consider doing one method and its test for each branch, and naming the branch for the method you wrote.